.vscode/settings.json

```diff
@@ -21,5 +21,8 @@
     "statusBar.foreground": "#15202b",
     "activityBar.activeBackground": "#66abe1"
   },
-  "peacock.color": "#3b94d9"
+  "peacock.color": "#3b94d9",
```

Comment:
Whoops. The .gitifnore in this file has a defect. XD

Reply:
I do love auto format ;)